src/views/blocks/rsvp/status/going.php

```diff
@@ -22,7 +22,7 @@
 	class="tribe-block__rsvp__status-button tribe-block__rsvp__status-button--going<?php if ( 'yes' === $going ) { echo ' tribe-active'; }?>"
 	<?php echo disabled( 'yes', $going, false ); ?>
 	>
-		<span><?php esc_html_e( 'Going', 'event-tickets' ); ?></span>
+		<span><?php echo esc_html_x( 'Going', 'Button label.', 'event-tickets' ); ?></span>
```

Comment:
I would suggest more context here, maybe something like "Label for the RSVP going button"